File: js/divisibilityProblem.js
function divisibilityProblem () {
    const testCaseArr = [85, 21,200,41];
    let num = "";
    testCaseArr.forEach((i) => {
        num = num + (i%10);
    })
    console.log(num);
    if(num % 5 == 0){
        console.log("Is divisible ");
    }else{
        console.log("Not divisible ");
    }
}

divisibilityProblem();

File: java/DivisibilityProblem.java
import java.util.Scanner;

public class DivisibilityProblem {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.println("Enter number of elements : ");
        int n = scan.nextInt();
        int [] elements = new int[n];
        for(int i = 0; i < n; i++){
            elements[i] = scan.nextInt();
        }
        String num = "";
        for(int i : elements) {
            int unitDigit = i % 10;
            num = num + unitDigit;
        }
        if(Integer.valueOf(num) % 5 ==0){
            System.out.println("Divisible by 5");
        }
        else {
            System.out.println("Not divisible by 5");
        }
    }
}